### DISCUSSION TITLE
Add h2 tag in Laravel Nova Trix

### DISCUSSION THREAD

**Discussion by hilmanski** [laravel/nova-issues#3754] [Q&A]
Hi, it's a question not a feature request.
I'd like to extend the trix editor to have several heading level such as H2, H3
I know that Trix is separate package, not part of Nova.

I just need a guidance on where to start in case anyone know,
on how to extend/hack the Trix editor on laravel Nova

thanks you everyon!

**Comment by sajaddp** [comment]
Hi, @hilmanski 
Did you solve this problem?
